src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/RawAtlasSlicer.java

```diff
@@ -1188,7 +1189,14 @@ private void sliceArea(final Line line)
             return;
         }
 
-        if (!checkSlices(slices.keySet(), slices.values().size(), line))
+        long numSlices = 0;
+        for (final Set<org.locationtech.jts.geom.Polygon> sliceSet : slices.values())
+        {
+            numSlices += sliceSet.size();
+        }
+        logger.warn("Way {} was sliced into {} slices", line.getOsmIdentifier(), numSlices);
```

Comment:
It seems this warn statement happens without check on the numSlices? Or is that snippet of code already in a corner-case section of code?

Reply:
in this case, maybe `INFO` is more appropriate. the 0 number is still useful to log, as entities with bounding boxes overlapping a boundary but *not* intersecting the boundary will run through this logic only to have no slicing actually happen-- this log allows for seeing those ways directly.